Automattic/jetpack
# 7.9: Changelog

#### Changes proposed in this Pull Request:

* Changelog and testing list for Jetpack 7.9

#### Testing instructions:

* Check for typos
* Check that everything reads well and is clear.

#### Proposed changelog entry for your changes:

* None
